Python tracing tests are failing

**Describe the bug**
Python tracing tests are failing because of some package error while installing from requirements.txt in utils/tracing/integ-tests.

**To Reproduce**
Run the test on any PR.

**Expected behavior**
Test should pass.

**Logs**
```
Run make all-image
cd ../../../../ && docker build --tag vhiveease/py-tracing-server:latest --target server -f ./utils/tracing/integ-tests/client-server/Dockerfile .
Sending build context to Docker daemon  10.75MB

Step 1/9 : FROM python:3.8-slim-buster as server
3.8-slim-buster: Pulling from library/python
a4b007099961: Pulling fs layer
66684bb0aed0: Pulling fs layer
e25203a91ebd: Pulling fs layer
513486c7a35b: Pulling fs layer
500882a508f7: Pulling fs layer
513486c7a35b: Waiting
500882a508f7: Waiting
66684bb0aed0: Verifying Checksum
66684bb0aed0: Download complete
e25203a91ebd: Verifying Checksum
e25203a91ebd: Download complete
a4b007099961: Verifying Checksum
a4b007099961: Download complete
513486c7a35b: Verifying Checksum
513486c7a35b: Download complete
500882a508f7: Verifying Checksum
500882a508f7: Download complete
a4b007099961: Pull complete
66684bb0aed0: Pull complete
e25203a91ebd: Pull complete
513486c7a35b: Pull complete
500882a508f7: Pull complete
Digest: sha256:b9c6d865d1a0e6c1a0c3e0ee7a15d37ac2e9d0a195c9c23d4ec5e6ccd6e06cc0
Status: Downloaded newer image for python:3.8-slim-buster
 ---> 970d54d92ed9
Step 2/9 : WORKDIR /app
 ---> Running in d30da34ab541
Removing intermediate container d30da34ab541
 ---> 9b8a7a9afa09
Step 3/9 : COPY ./utils/tracing/integ-tests/client-server/requirements.txt .
 ---> 5906f6379809
Step 4/9 : COPY ./utils/tracing/integ-tests/client-server/*.py ./
 ---> 606bb516a93f
Step 5/9 : COPY ./utils/tracing/python/tracing.py .
 ---> 0527a271cb34
Step 6/9 : RUN pip3 install --user -r requirements.txt
 ---> Running in 5bb2bf70c489
Collecting futures
  Downloading futures-3.0.5.tar.gz (25 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [25 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 14, in <module>
        File "/usr/local/lib/python3.8/site-packages/setuptools/__init__.py", line 18, in <module>
          from setuptools.dist import Distribution
        File "/usr/local/lib/python3.8/site-packages/setuptools/dist.py", line 32, in <module>
          from setuptools.extern.more_itertools import unique_everseen
        File "<frozen importlib._bootstrap>", line 991, in _find_and_load
        File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
        File "<frozen importlib._bootstrap>", line 657, in _load_unlocked
        File "<frozen importlib._bootstrap>", line 556, in module_from_spec
        File "/usr/local/lib/python3.8/site-packages/setuptools/extern/__init__.py", line 52, in create_module
          return self.load_module(spec.name)
        File "/usr/local/lib/python3.8/site-packages/setuptools/extern/__init__.py", line 37, in load_module
          __import__(extant)
        File "/usr/local/lib/python3.8/site-packages/setuptools/_vendor/more_itertools/__init__.py", line 1, in <module>
          from .more import *  # noqa
        File "/usr/local/lib/python3.8/site-packages/setuptools/_vendor/more_itertools/more.py", line 5, in <module>
          from concurrent.futures import ThreadPoolExecutor
        File "/tmp/pip-install-9vnqstn1/futures_7a94e9de7c984882b7827a2c23da7a9c/concurrent/futures/__init__.py", line 8, in <module>
          from concurrent.futures._base import (FIRST_COMPLETED,
        File "/tmp/pip-install-9vnqstn1/futures_7a94e9de7c984882b7827a2c23da7a9c/concurrent/futures/_base.py", line 357
          raise type(self._exception), self._exception, self._traceback
                                     ^
      SyntaxError: invalid syntax
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
The command '/bin/sh -c pip3 install --user -r requirements.txt' returned a non-zero code: 1
make: *** [server-image] Error 1
Makefile:28: recipe for target 'server-image' failed
Error: Process completed with exit code 2.
```
[Link to one of the PR where it fails](https://github.com/ease-lab/vSwarm/runs/5604158267?check_suite_focus=true)

--- a/utils/tracing/integ-tests/client-server/requirements.txt
+++ b/utils/tracing/integ-tests/client-server/requirements.txt
@@ -1,4 +1,3 @@
-futures
 grpcio==1.38.1
 opentelemetry-api==1.3.0
 opentelemetry-exporter-zipkin==1.3.0

